### PR TITLE
Academic grants page & form integration

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds the Academic grants page & form

## Description

- adds Academic Grants page (`/academic-grants`)
- adds Academic Grants form (`/academic-grants/apply`)
- adds Academic Grants thank you page (`/academic-grants/thank-you`)
- adds SF API integration
- updates `.env.local.example`
- minor UI fixes

### CODE REVIEW

File: src/components/forms/constants.ts [hunk 1]
```diff
@@ -1434,6 +1454,21 @@ export const HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS = [
...
 
+export const HOW_DID_YOU_HEAR_ABOUT_GRANTS_WAVE = [
+  { value: 'Email received', label: 'Email received' },
+  ...HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS
+];
+
+// academic grants
+export const APPLYING_AS_OPTIONS = [
+  { value: 'An individual', label: 'An individual' },
+  { value: 'A University	', label: 'A University	' },
```
Comment (pettinarip) [inline src/components/forms/constants.ts:1465]: Any reason for that space in the value and label?

**Review by pettinarip — APPROVED** [review]
LGTM!

File: src/pages/academic-grants/index.tsx [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+                        proposal that advances the Ethereum ecosystem, we encourage you to head over
+                        to ESP.
+                      </PageText>
+                    </FAQItem>
+                  </Accordion>
+                </section>
+
+                <section id='apply' ref={ref9}>
+                  <Stack mt={6}>
+                    <ReadyToApply link={`${ACADEMIC_GRANTS_URL}/apply`} />
```
[the remark this answers is not in the document]
Comment (nhsz) [reply src/pages/academic-grants/index.tsx:348]: fixed!

File: src/components/forms/constants.ts (hunk 1 above)
Comment (nhsz) [reply src/components/forms/constants.ts]: No reason, it's a typo, fixed now!

**Review by pettinarip — APPROVED** [review]

**Pull request merged**